File: module4/practical/p8.cpp
// Write a C++ program that takes a student’s marks as input and calculates the grade 
// based on if-else conditions. 
#include<iostream>
using namespace std;
int main(){
    int marks;
    cout<<"Enter the marks "<<endl;
    cin>>marks;

    if(marks>=90){
        cout<<"A"<<endl;
    }else if(marks>=75 && marks<90){
        cout<<"B"<<endl;
    }else if(marks>=60 && marks<75 ){
        cout<<"C"<<endl;
    }else if(marks>=45 && marks<60){
        cout<<"D"<<endl;
    }else{
        cout<<"Fail";
    }


}

File: module7/practical/p8.py
# Write a Python program to create a function that takes a string as input and prints it. 

def user_input(str):
    print(str)

user_input("python")


# • Write a Python program to create a calculator using functions. 


def calculator(num1,num2,op):

    if op=='+':
        print(f"Addition is {num1+num2}")
    elif op=='-':
        print(f"Subtraction is {num1-num2}")
    elif op=='*':
        print(f"Multiplication is {num1*num2}")
    elif op=='/':
        print(f"Division  is {num1/num2}")
    elif op=='%':
        print(f"Modulas is {num1%num2}")
    else:
        print("Invalid operation ")


num1=int(input("enter the num1 ::"))
num2=int(input("enter the num1 ::"))
op=input("enter the operation")

calculator(num1,num2,op)